valhalla/mjolnir/osmway.h

```diff
@@ -2679,9 +2727,15 @@ struct OSMWay {
   uint16_t use_sidepath_ : 1;
   uint16_t bike_forward_ : 1;
   uint16_t bike_backward_ : 1;
+  // Flags which didn't fit elsewhere
   uint16_t lit_ : 1;
   uint16_t destination_only_hgv_ : 1;
-  uint16_t spare2_ : 2;
+  // 2 access flags that didn't fit in the previous 16 bits
+  uint16_t golf_cart_forward_ : 1;
+  uint16_t golf_cart_backward_ : 1;
+
+  // Sadly this doesn't fit packed into the previous 16 bits so it's on its own for now
+  bool golf_cart_designated_;
```

Comment:
Worth noting: I added 3 bits in total, which means overflowing the existing nicely packed set of fields :(

Reply:
bool wont become a bit. the compiler will pad that to either 1, 4 or 8 bytes depending on your platform. to avoid confusion we should probably just add another `uint16_t` or another type with a specific width (whatever makes sense depending on what we already have in here to get to a round number (multiple of 64 bits)) and then take 1 bit from it and make the rest spare.

Reply:
That's a good point for me to follow up on.

The people actually using this stated that they wanted it to be able to ensure that golf carts used appropriate paths even when another road was legally available, but doing a quick check of the areas with overpass turbo, I'm not actually sure they are :P If that's the case, we could possibly free up that bit + simplify the logic a bit.

Reply:
i was hoping you might say that :smile:

Reply:
Followed up with the relevant parties and they are on board. I've removed all the designated logic.